src/lib.rs

```diff
@@ -345,13 +363,41 @@ impl From<Cuboid> for FocusBoundsShape {
     }
 }
 
+/// Allows for changing the TrackpadBehavior from default to the way it works in Blender.
+/// In Blender the trackpad orbits when scrolling. If you hold down the ShiftLeft, it Pans and
+/// holding down ControlLeft will Zoom.
+#[derive(Clone, PartialEq, Debug, Reflect, Copy)]
+pub enum TrackpadBehavior {
+    /// Default bevy_panorbit_camera behavior is to zoom
+    DefaultZoom,
```

Comment:
I think the naming is a bit confusing. The touchpad does more than just zoom. Let's change it to this:

```suggestion
    /// Default touchpad behavior. I.e., no special gesture support, scrolling on the touchpad (vertically) will zoom, as it does with a mouse.
    Default,
```